pkg/operator/controller/instance_controller.go

```diff
@@ -65,6 +65,7 @@ type InstanceReconciler struct {
 // +kubebuilder:rbac:groups=clusterlink.net,resources=instances,verbs=list;get;watch;update;patch
 // +kubebuilder:rbac:groups=clusterlink.net,resources=instances/status,verbs=get;update;patch
 // +kubebuilder:rbac:groups=clusterlink.net,resources=instances/finalizers,verbs=update
+// +kubebuilder:rbac:groups="",resources=secrets,verbs=list;get;watch;create;update
```

Comment:
nit: is there a need for Cluster scope or can we create a Namespaced role for reading just the secret in our namespace?
you can also further limit using resourceNames in the kubebuilder annotation

Reply:
Although by default the namespace is "clusterlink-system", it is customizable.
So I don't think we can use the namespace annotation.
Also the resource name is defined in a go const string. I don't see how can I refer to it in the kubebuilder comment.